app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt

```diff
@@ -0,0 +1,166 @@
+package jp.developer.retia.frozenword.ui.watch
+
+import androidx.annotation.IntRange
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import jp.developer.retia.frozenword.ui.theme.FrozenWordTheme
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+@Preview
+@Composable
+fun PreviewHabbits() {
+    FrozenWordTheme {
+        Surface {
+            WatchScreen(
+                title = "運動",
+                simpleHabbitTitle = "着替える",
+                trigger = "朝起きたら",
+                place = "自室で"
+            )
+        }
+    }
+}
+
+@Composable
+fun WatchScreen(
+    title: String,
+    simpleHabbitTitle: String,
+    trigger: String? = null,
+    place: String? = null,
+    modifier: Modifier = Modifier,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/69693efc910066613382c0b4d00078525306e025/app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt#L52"></span>Modifier parameter should be the first optional parameter</td>
    </tr>
  </tbody>
</table>